whois-query/src/main/java/net/ripe/db/whois/query/hazelcast/HazelcastInstanceManager.java

```diff
@@ -0,0 +1,55 @@
+package net.ripe.db.whois.query.hazelcast;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import net.ripe.db.whois.common.profiles.DeployedProfile;
+import net.ripe.db.whois.common.profiles.WhoisProfile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+@DeployedProfile
+public class HazelcastInstanceManager {
+    private static final Logger LOGGER = LoggerFactory.getLogger(HazelcastInstanceManager.class);
+
+    final String interfaces;
+
+    @Autowired
+    public HazelcastInstanceManager( @Value("${hazelcast.config.interaces:10.*.*.*}") final String interfaces) {
```

Comment:
Is there a typo in the property name? ```hazelcast.config.interaces``` (interfaces?)

Reply:
Yes there is